### DISCUSSION TITLE
IBS-TH1 ext probe struggle for home assistant

### DISCUSSION THREAD

**Discussion by checkiecheck** [theengs/gateway#142] [Q&A]
I've got the IBS th1 with the external probe attached. With the current setup to homeassistant it has state_class 'measurement' but it does only report true/false. With this, the home assistant logs get spammed with 
ValueError: Sensor sensor.ibs_th1_th2_p01b_extprobe has device class 'None', state class 'measurement' unit 'None' and suggested precision 'None' thus indicating it has a numeric value; however, it has the non-numeric value: 'True' (<class 'str'>)

If i understand well, it's the decoder that does this, but i cant see how to correct it.
How can i fix this from the gateway or somewhere else?

**Comment by 1technophile** [comment]
I think an exception could be done at gateway level for the sensor with the key extprobe, so that they don't get the class `measurement` assigned
https://github.com/theengs/gateway/blob/156a46ae5bd596209edaf6660f0248bea6013f5f/TheengsGateway/discovery.py#L149
